packages/docusaurus-theme-classic/src/theme/LastUpdated/index.tsx

```diff
@@ -34,7 +34,7 @@ function LastUpdatedAtDate({
       values={{
         date: (
           <b>
-            <time dateTime={atDate.toISOString()}>
```

Comment:
This is adding the date to microdata, but we also need to add it to JSON LD structured data 

cc @johnnyreilly we are a bit in a weird place IMHO because only the blog plugin uses JSON LD atm, and we have components shared between docs and blog 😅 

I guess it's fine if we keep this change, and the blog will have both the structured data as microdate + json ld?

Reply:
Let's try it I think